### DISCUSSION TITLE
roboshop components error frontend catalogue user

### DISCUSSION THREAD

**Discussion by raghavarohith** [learndevopsonline/learndevopsonline#984] [Q&A]
![user](https://github.com/learndevopsonline/learndevopsonline/assets/85346264/274385bd-8e4d-4ad2-b2a7-09f9ea8669e5)
![catalogue](https://github.com/learndevopsonline/learndevopsonline/assets/85346264/d6c9546c-0cca-4c04-8727-0a9f343e5603)
![mongodb](https://github.com/learndevopsonline/learndevopsonline/assets/85346264/2a4b3437-f855-4f36-8b1d-7ac9b4fb5031)

**Comment by raghavarohith** [comment]
frontend ip address 44.201.4.108

**Reply by Aleem410** [reply]
please check now
![Uploading roboshop.PNG…]()

**Reply by Aleem410** [reply]
<img width="869" alt="roboshop" src="https://github.com/learndevopsonline/learndevopsonline/assets/118476673/6b69b930-418e-43aa-b158-818f04e20068">

**Comment by Aleem410** [comment]
<img width="866" alt="user" src="https://github.com/learndevopsonline/learndevopsonline/assets/118476673/0bfdb51b-a89c-4ca9-867b-7e3e4bcb6b6c">
